internal/transform/transaction.go

```diff
@@ -69,12 +69,18 @@ func TransformTransaction(transaction ingest.LedgerTransaction, lhe xdr.LedgerHe
 	timeBound := transaction.Envelope.TimeBounds()
 	outputTimeBounds := ""
 	if timeBound != nil {
-		if timeBound.MaxTime < timeBound.MinTime {
+		if timeBound.MaxTime < timeBound.MinTime && timeBound.MaxTime > 0 {
```

Comment:
💡 not sure if this matters but the case of timeBound.MaxTime being negative value isn't caught. Not sure if that's a possible value, but just to be safe this line could be:

```suggestion
		if timeBound.MaxTime < timeBound.MinTime && timeBound.MaxTime != 0 {
```

alternatively could make the line a couple lines down:
`if timeBound.MaxTime <= 0 {`

Reply:
re-reading your description it doesn't look negative is a possible value, so you can ignore this comment if you don't think the change is needed

Reply:
Committing the suggestion just in case there is an exception in the future